doc/commands.doc

```diff
@@ -605,7 +605,8 @@ Structural indicators
 
   \par Important:
   The documentation of global functions, variables, typedefs, and enums will
-  only be included in the output if the file they are in is documented as well.
+  only be included in the output if the file they are in is documented as well
+  or if \ref cfg_javadoc_autobrief "EXTRACT_ALL" is set to \c YES.
```

Comment:
The construct ` \ref cfg_javadoc_autobrief "EXTRACT_ALL"` is very strange as the text is `EXTRACT_ALL` but the jump is to `JAVADOC_AUTOBRIEF`.

Is the intention:
  - `\ref cfg_javadoc_autobrief "JAVADOC_AUTOBRIEF"`

or
  - `\ref cfg_extract_all "EXTRACT_ALL"`

or 
  - ...